dcos/marathon.py

```diff
@@ -109,8 +109,9 @@ def response_error_message(cls, status_code, reason, request_method,
             return template.format(request_method, request_url, reason, suffix)
 
         if status_code == 409:
-            return ('App, group, or pod is locked by one or more deployments. '
-                    'Override with --force.')
+            return ('App, group, or pod already exists and cannot be changed '
+                    'at this time. Try again later or use --force if it is'
+                    'available for your command.')
```

Comment:
Lets keep it simple: `App, group, or pod already exists`. Users shouldnt be using force if they dont know what it'll do. 

We can even be fancier, and since we have the `request_method` tell the user <i>which</i> resource already exists.